contracts/src/finance/vesting_wallet.rs

```diff
@@ -81,8 +85,9 @@ pub enum Error {
 }
 
 pub use token::IErc20;
-#[allow(missing_docs)]
 mod token {
+    #![allow(missing_docs)]
```

Comment:
You cannot apply this attr. to the `sol_interface` macro. Running `cargo check` locally produces the warning that this code is missing docs.

Worryingly, there seem to be [no issues in GH actions](https://github.com/OpenZeppelin/rust-contracts-stylus/actions/runs/12225648823/job/34099905478#step:5:1) :fearful:

Reply:
`#![allow(missing_docs)]` I have applied it modulewise. You should see `!` between '#' and `[`

Reply:
You're right, so the behavior is the same as before